src/main/java/com/minecolonies/coremod/blocks/BlockConstructionTape.java

```diff
@@ -0,0 +1,281 @@
+
+    /**
+     * The hardness this block has.
+     */
+    private static final float BLOCK_HARDNESS = 0.0F;
+
+    /**
+     * This blocks name.
+     */
+    private static final String BLOCK_NAME = "blockConstructionTape";
+
+    /**
+     * The resistance this block has.
+     */
+    private static final float RESISTANCE = 0.0F;
+
+    /**
+     * Start of the collision box at y.
+     */
+    private static final double BOTTOM_COLLISION = 0.0;
+
+    /**
+     * Start of the collision box at x facing South/North.
+     */
+    private static final double SN_START_COLLISION_X = 0.0;
+
+    /**
+     * End of the collision box facing South/North.
+     */
+    private static final double SN_END_COLLISION_X = 1.0;
+
+    /**
+     * Start of the collision box at z facing South/North.
+     */
+    private static final double SN_START_COLLISION_Z = 0.4375;
+
+    /**
+     * End of the collision box facing South/North.
+     */
+    private static final double SN_END_COLLISION_Z   = 0.5625;
+    /**
+     * Start of the collision box at x facing South/North.
+     */
+    private static final double WE_START_COLLISION_X = 0.4375;
+
+    /**
+     * End of the collision box facing South/North.
+     */
+    private static final double WE_END_COLLISION_X = 0.5625;
+
+    /**
+     * Start of the collision box at z facing South/North.
+     */
+    private static final double WE_START_COLLISION_Z = 0.0;
+
+    /**
+     * End of the collision box facing South/North.
+     */
+    private static final double WE_END_COLLISION_Z = 1.0;
+
+    /**
+     * Height of the collision box.
+     */
+    private static final double HEIGHT_COLLISION = 1.0;
+    /**
+     * How much light goes through the block.
+     */
+    private static final int    LIGHT_OPACITY    = 0;
+
+    /**
+     * Constructor for the Substitution block.
+     * sets the creative tab, as well as the resistance and the hardness.
+     */
+    public BlockConstructionTape()
+    {
+        super(Material.VINE);
+        initBlock();
+    }
+
+    /**
+     * initialize the block
+     * sets the creative tab, as well as the resistance and the hardness.
+     */
+    private void initBlock()
+    {
+        setRegistryName(BLOCK_NAME);
+        setUnlocalizedName(String.format("%s.%s", Constants.MOD_ID.toLowerCase(), BLOCK_NAME));
+        setCreativeTab(ModCreativeTabs.MINECOLONIES);
+        this.setDefaultState(this.blockState.getBaseState().withProperty(FACING, EnumFacing.NORTH));
+        setHardness(BLOCK_HARDNESS);
+        setResistance(RESISTANCE);
+        setLightOpacity(LIGHT_OPACITY);
+    }
+
+    /**
+     * Convert the given metadata into a BlockState for this Block.
+     *
+     * @deprecated (Remove this as soon as minecraft offers anything better).
+     */
+    @SuppressWarnings(DEPRECATION)
+    @NotNull
+    @Override
+    @Deprecated
+    public IBlockState getStateFromMeta(final int meta)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1133)